buzsaki_lab_to_nwb/tingley_code_septal/tingleyseptalbehaviorinterface.py

```diff
@@ -0,0 +1,167 @@
+        if units == "m":
+            conversion = 1.0
+        else:
+            warnings.warn(f"Spatial units {units} not listed in meters; " "setting conversion to nan.")
+            conversion = np.nan
+
+        description = behavior_mat.get("description", "generic_position_tracking").replace("/", "-")
+        rotation_type = behavior_mat.get("rotationType", "non_specified")
+
+        pos_obj = Position(name=f"{description}")
+
+        spatial_series_object = SpatialSeries(
+            name=f"Position",
+            description=f"(x,y,z) coordinates tracking subject movement through",
+            data=H5DataIO(pos_data, compression="gzip"),
+            reference_frame="unknown",
+            conversion=conversion,
+            timestamps=timestamps,
+            resolution=np.nan,
+        )
+
+        pos_obj.add_spatial_series(spatial_series_object)
+
+        # Add error if available
+        errorPerMarker = behavior_mat.get("errorPerMarker", None)
+        if errorPerMarker:
+            error_data = np.array([error for error in errorPerMarker])[..., 0]
+
+            spatial_series_object = SpatialSeries(
+                name=f"Error per marker",
+                description=f"Error per marker",
+                data=H5DataIO(error_data, compression="gzip"),
+                reference_frame="unknown",
+                conversion=conversion,
+                timestamps=timestamps,
+                resolution=np.nan,
+            )
+            pos_obj.add_spatial_series(spatial_series_object)
+
+        processing_module.add_data_interface(pos_obj)
+
+        # Compass
+        try:
+            orientation = behavior_mat["orientation"]
+            orientation_data = [
+                [x, y, z, w]
+                for (x, y, z, w) in zip(orientation["x"], orientation["y"], orientation["z"], orientation["w"])
+            ]
+            orientation_data = np.array(orientation_data)[..., 0]
+
+            module_name = "Orientation"
+            module_description = "Contains behavioral data concerning orientation."
+            processing_module = get_module(nwbfile=nwbfile, name=module_name, description=module_description)
+
+            compass_obj = CompassDirection(name=f"route centric")
+
+            spatial_series_object = SpatialSeries(
+                name=f"Orientation",
+                description=f"(x, y, z, w) orientation coordinates, orientation type: {rotation_type}",
+                data=H5DataIO(orientation_data, compression="gzip"),
+                reference_frame="unknown",
+                conversion=conversion,
+                timestamps=timestamps,
+                resolution=np.nan,
+            )
+            compass_obj.add_spatial_series(spatial_series_object)
+            processing_module.add_data_interface(compass_obj)
+
+        except KeyError:
+            warnings.warn(f"Orientation data not found")
+
+        # States
+        module_name = "Neural states"
```

Comment:
```suggestion
        module_name = "neural_states"
```
Reverting back to using underscores for all naming conventions now